src/common/config.cc

```diff
@@ -604,15 +604,19 @@ int md_config_t::parse_option(std::vector<const char*>& args,
 
   if (ret != 0 || !error_message.empty()) {
     if (oss) {
-      *oss << "Parse error setting " << option_name << " to '"
-           << val << "' using injectargs";
+      if(option_name && !val.empty()) {
```

Comment:
@tchaikov Yes you are right. It happens only if it doesn't enter the loop:

`  for (auto& opt_ref: *config_options) {`

, which might be a rare case. But this check is good to silence the Coverity Warning and that's the main intention here.